src/pkg/services/m365/api/lists_test.go

```diff
@@ -68,6 +69,389 @@ func (suite *ListsUnitSuite) TestBytesToListable() {
 	}
 }
 
+func (suite *ListsUnitSuite) TestColumnDefinitionValidationSetWhenProvided() {
+	tests := []struct {
+		name     string
+		getOrig  func() models.ColumnDefinitionable
+		checkNil bool
```

Comment:
you can pack the assert call into the property itself here.
```suggestion
		expect assert.ValueAssertionFunc
```

This lets you change the test value to:
```go
expect: assert.Nil // or assert.NotNil
```

and verify the result like so
```go
newCd := cloneColumnDefinitionable(orig)
require.NotEmpty(t, newCd)
test.expect(t, newCd.GetValidation())
```